phoebe/frontend/default_bundles/default_contact_binary.bundle

```diff
@@ -2871,7 +2866,7 @@ null
 "qualifier": "phoebe_version",
 "context": "setting",
 "description": "Version of PHOEBE",
-"value": "2.4.8.dev+feature-blending",
+"value": "2.4.3.dev+feature-blending",
```

Comment:
this looks like a reversion - can you recreate the default bundles just to make sure they're up to date with the new/revised blending parameters?  And we'll have to decide whether tmap does or does not belong in the list of atmospheres at this point.

Reply:
recreated, tests running.